### DISCUSSION TITLE
regexManager + bumpVersion

### DISCUSSION THREAD

**Discussion by jkroepke** [renovatebot/renovate#19613] [Request Help]
Hi,

I'm currently using renovate to bump my docker images inside Chart.yaml using a regex manager (https://github.com/jkroepke/helm-charts/blob/main/renovate.json).

Now I want to use the post task [bumpVersion](https://docs.renovatebot.com/configuration-options/#bumpversion) to bump the helm chart.yaml version. It seems like its not compatible with regexManager. Is there an option the forcefully use the helmv3 bumpVersion option?

**Comment by viceice** [comment]
no
